doc/api/_toc.md

```diff
@@ -14,6 +14,7 @@
 * [Console](console.html)
 * [Crypto](crypto.html)
 * [Debugger](debugger.html)
+* [Deprecated APIs](deprecations.html)
```

Comment:
nit: maybe use the same word in the file name and the section name? I.e. `deprecated.md/html` or even `deprecated-apis.md/html`?